libraries/libxml/source/org/pentaho/reporting/libraries/xmlns/writer/XmlWriterSupport.java

```diff
@@ -794,14 +803,19 @@ public void writeTextNormalized( final Writer writer, final String s, final bool
    * @param transformNewLine true, if a newline in the string should be converted into a character entity.
    * @return the normalised string.
    */
-  public String normalize( final String s, final boolean transformNewLine ) throws IOException {
+  public static String normalize( final String s, final boolean transformNewLine ) {
```

Comment:
Please remove the "static". Within a point release you are not allowed to change the public API of existing classes. Making a method static is such a breaking change, and worst of all, it will break all jars that are compiled against older versions of the library.